pkgs/by-name/nh/nhentai/package.nix

```
@@ -1,31 +1,67 @@
 {
   lib,
+  python3,
   python3Packages,
   fetchFromGitHub,
 }:
 
+let
+  python =
+    let
+      packageOverrides = self: super: {
+        iso8601 = super.iso8601.overridePythonAttrs (old: rec {
+          version = "1.1.0";
+          src = super.fetchPypi {
+            pname = "iso8601";
+            inherit version;
+            hash = "sha256-MoEee4He7iBj6m0ulPiBmobR84EeSdI2I6QfqDK+8D8=";
+          };
+        });
+        urllib3 = super.urllib3.overridePythonAttrs (old: rec {
+          version = "1.26.20";
+          src = super.fetchPypi {
+            pname = "urllib3";
+            inherit version;
+            hash = "sha256-QMLcDGgeR+uPkOfie/b/ffLmd0If1GdW2hFhw5ynDTI=";
+          };
+        });
+      };
+    in
+    python3.override {
+      inherit packageOverrides;
+      self = python;
+    };
+
+in
 python3Packages.buildPythonApplication rec {
```

```suggestion
python.pkgs.buildPythonApplication rec {
```
Imho this'd be more consistent + should reduce the closure size